ContactsListVer2/ContactAddEditPresenter.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  ContactAddEditPresenter.swift
+//  ContactsListVer2
+//
+//  Created by Maksym Korostelov on 5/30/17.
+//  Copyright © 2017 Maksym Korostelov. All rights reserved.
+//
+
+import Foundation
+
+protocol ContactAddEditPresenterProtocol: class {
+    init(contactAddEditVC: ContactAddEditProtocol, contactList: ContactsList?, contactUuid: String?)
+    
+    func viewDidLoad()
```

Comment:
Avoid duplication of methods signatures with library (using of viewDidLoad on non-UI object looks weird and may confuse other developers)